src/wallet/rpcwallet.cpp

```diff
@@ -2382,6 +2382,7 @@ static UniValue getbalances(const JSONRPCRequest& request)
                     {RPCResult::Type::STR_AMOUNT, "untrusted_pending", "untrusted pending balance (outputs created by others that are in the mempool)"},
                     {RPCResult::Type::STR_AMOUNT, "immature", "balance from immature coinbase outputs"},
                 }},
+                {RPCResult::Type::STR_HEX, "lastblock", "The hash of the block the balances were valid on"},
```

Comment:
calculated?

Reply:
Yes, this too was changed when I converted `lastblock`into an object that contains both hash and height.